nbconvert/filters/markdown_mistune.py

```diff
@@ -106,6 +106,11 @@ def output_latex_environment(self, m):
 
 class MarkdownWithMath(mistune.Markdown):
     def __init__(self, renderer, **kwargs):
+        # This makes mistune behave more like markedjs
+        # (markedjs behing used by Jupyter Notebook and JupyterLab)
```

Comment:
Small typo in `behing` :wink: 

Very nice find.

Reply:
Thanks!